src/Makefile

```diff
@@ -9,13 +9,13 @@ init:
 env=remote
 export DJANGO_ENV=$(env)
 start:
-	python deployment/prepare_to_deploy.py local 1
-	python manage.py runserver
+	python3 deployment/prepare_to_deploy.py local 1
```

Comment:
changing all these to python3 for consistency of all our developers.  Also it makes it easier for the github action to find the right python version we want and hence the right pip packages.

Reply:
100%!